integration_tests/junit-vintage-engine/build.gradle.kts

```
@@ -0,0 +1,50 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+  alias(libs.plugins.android.library)
+  alias(libs.plugins.detekt)
+  alias(libs.plugins.kotlin.android)
+  alias(libs.plugins.robolectric.android.project)
+  alias(libs.plugins.robolectric.spotless)
+}
+
+android {
+  namespace = "org.robolectric.integrationtests.junit.vintage"
+  compileSdk = 36
+
+  defaultConfig { minSdk = 23 }
+
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+  }
+
+  testOptions {
+    targetSdk = 36
+    unitTests.isIncludeAndroidResources = true
+  }
+
+  androidComponents {
+    beforeVariants { variantBuilder ->
+      // junit-vintage-engine test module does not support AndroidTest now.
+      variantBuilder.enableAndroidTest = false
+    }
+  }
+}
+
+tasks.withType(Test::class.java) { useJUnitPlatform() }
```

You should be able to do one of the following:

```suggestion
tasks.withType<Test> { useJUnitPlatform() }
```

Or:

```suggestion
tasks.tests { useJUnitPlatform() }
```